pt-BR/mail/chrome/messenger/safeMode.dtd

```diff
@@ -72,7 +71,7 @@ entities below were automatically added by l10nmerge
 
 <!ENTITY resetImapFolders.accesskey       "b">
 <!ENTITY resetCacheAndIndex.accesskey     "I">
-<!ENTITY resetImapFolders.label           "Reset IMAP mail database">
-<!ENTITY resetCacheAndIndex.label         "Reset Caches and Search Index">
-<!ENTITY safeModeDialogHelpButton.tooltip "Display online help for this dialog">
-<!ENTITY safeModeDialogHeader.label       "Troubleshooting">
+<!ENTITY resetImapFolders.label           "Redefinir a base de dados de e-mails IMAP">
+<!ENTITY resetCacheAndIndex.label         "Redefinir os caches e o índice de pesquisas">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY resetCacheAndIndex.label         "Apagar os dados temporários e o índice de pesquisas">